docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc

```diff
@@ -92,6 +92,27 @@ To disable this behavior, use the SPI option `spi-single-use-object-infinispan-p
 The SPI behavior of `SingleUseObjectProvider` has changed that for revoked tokens only the methods `put` and `contains` must be used.
 This is enforced by default, and can be disabled using the SPI option `spi-single-use-object-infinispan-persist-revoked-tokens`.
 
+= HA Multi-Site Deployments
+
+{project_name} 26 introduces significant improvements to the recommended HA multi-site architecture, most notably:
+
+- {project_name} deployments on each site are now able to handle user requests simulaneously, Active/Active vs the previous
+Active/Passive architecture.
+- LoadBalancing is now handled by AWS Global Accelerator, meaning there's no longer issues with DNS caching on failover/switchover
+that existed with the previous Route53 architecture.
+- {project_name} utilises Infinispan RemoteCache's to communicate with Infinispan deployments more efficiently.
+
+As a consequence of the above changes, the following changes are required to your existing {project_name} deployments.
+
+. `distributed-cache` definitions provided by `--cache-config-file` are ignored when the `multi-site` feature is enabled,
+so you must configure the connection to the external Infinispan deployment via the `cache-remote-*` CLI args. If a
```

Comment:
TODO: The `advanced-configuration.adoc` doesn't detail on how to configure caches using the Keycloak CR. This might be a follow-up issue if you choose not to add it in this PR. 

```suggestion
so you must configure the connection to the external Infinispan deployment via the `cache-remote-*` command line arguments or Keycloak CR as outlined in the blueprints.  If a
```

Reply:
Currently the `advanced-configuration.adoc` seems to focus on providing generic guidance on the CRs capabilities, not use-case specific instructions. In the case of multi-site deployments we need to utilise the `spec.additionalOptions` and `spec.features` fields both of which are covered in the guide.

Looking at the Keycloak CRD, the only capability exposed by `spec.cache` is the ability to provide a cache config file via a ConfigMap, which in the case of multi-site deployments is no longer relevant for the common use-case as we have the `cache-remote-*` options.

Or have I missed the point and you have something else in mind?

Reply:
Could relate to #30217 - which would put more first class options in the CR.